### PR TITLE
[docs] Update STYLES.md re testing the changes

### PR DESCRIPTION
**Pull request by pastk**

### CODE REVIEW

**Review by vng — APPROVED** [review]

File: docs/STYLES.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ To work with styles first [clone the OM repository](INSTALL.md#getting-sources).
 To run some scripts additional libraries are needed:
 ```
 optipng
-protobuf 3.17.3 (3.18.0 doesn't work!)
+protobuf (a python package)
```
Comment (biodranik) [inline docs/STYLES.md:23]: How to install it?

File: docs/STYLES.md [hunk 2]
```diff
@@ -46,6 +46,7 @@ preferably look for icons in [collections OM uses already](../data/copyright.htm
 2. Add icon rendering/visibility rules into `data/styles/clear/include/Icons.mapcss` and to "navigation style" `data/styles/vehicle/include/Icons.mapcss`
 3. Run `tools/unix/generate_symbols.sh` to add new icons into skin files
 4. Run `tools/unix/generate_drules.sh` to generate drawing rules for the new icons
+5. Test the changes
```
Comment (biodranik) [inline docs/STYLES.md:49]:
```suggestion
5. [Test](#testing-your-changes) your changes
```

File: docs/STYLES.md [hunk 3]
```diff
@@ -57,21 +58,36 @@ preferably look for icons in [collections OM uses already](../data/copyright.htm
 6. Add search keywords into `data/categories.txt`
 7. Run `tools/unix/generate_localizations.sh` to validate and distribute translations into iOS and Android
 8. Add new or fix current classifier tests at `/generator/generator_tests/osm_type_tests.cpp` if you can
-9. Ideally, test the changes locally by [generating](MAPS.md) some small region map
+9. Test the changes
```
Comment (biodranik) [inline docs/STYLES.md:61]: [Test](#testing-your-changes) your changes

File: docs/STYLES.md (hunk 1 above)
Comment (pastk) [reply docs/STYLES.md]: Updated

**Review by biodranik — APPROVED** [review]

**Pull request merged**